lego/apps/notifications/templates/notifications/push/announcement.txt

```diff
@@ -1 +1 @@
-Du har mottatt en viktig melding fra {{created_by}}.
+Du har mottatt en viktig melding fra {% if from_group %} {{ from_group }} {% else %} {{ created_by }} {% endif %}.
```

Comment:
Maybe it would be cleaner to have a template variable `sender` here, so you could keep logic about who is the visible sender in the code. Just like you do with the email subject line.